layouts/_default/baseof.html

```diff
@@ -23,5 +23,8 @@
     {{ block "sidebar-footer" . }}{{ end }}
     {{ partial "footer/footer.html" . }}
     {{ partial "footer/script-footer.html" . }}
+    <!-- Start of blubracket Zendesk Widget script -->
+<script id="ze-snippet" src="https://static.zdassets.com/ekr/snippet.js?key=6aa6f6f6-af7c-4e14-834b-fb23c358fc25"> </script>
```

Comment:
Let's create a new file called `zendesk.html` in https://github.com/BluBracket/docs.blubracket.com/tree/main/layouts/partials/footer

Use the pattern in https://github.com/BluBracket/docs.blubracket.com/blob/main/layouts/partials/head/analytics.html to make the key configurable.

The key should be set in https://github.com/BluBracket/docs.blubracket.com/blob/main/config/production/config.toml